### PR TITLE
Add IE Banner

### PR DESCRIPTION
**Pull request by bitliege**
Sets up static content for #4579

We'll want to ensure some of the discussion in the issue is addressed. Ping me if there are questions!

### CODE REVIEW

File: resources/views/common/h2-ie-dialog.html.twig [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+          type="button">
+          <span>x</span>
+        </button>
+      </div>
+      <div
+        data-h2-dialog-content
+        data-h2-padding="b(all, 1)">
+        <div id="ieDialogContent">
+          <p>Thanks for visiting Talent Cloud!</p>
+          <p data-h2-margin="b(tb, 1)">As <a href="" title="">Microsoft has begun phasing out support for Internet Explorer</a>, we recommend using Talent Cloud from a modern web browser for the best experience. We recommend looking at any of the following:</p>
```
Comment (mnigh) [inline resources/views/common/h2-ie-dialog.html.twig:32]: @bitliege Should the `href` and `title` attributes in the `a` tag not have values or is this forthcoming?

**Comment by mnigh** [comment]
@tristan-orourke We will need translations for [Internet Explorer Banner and Dialog Language Lines](https://github.com/GCTC-NTGC/TalentCloud/blob/4b1dcadce72d88983fb0d22893e430d6459a7bee/resources/lang/fr/common/internet_explorer.php).

**Review by tristan-orourke — CHANGES_REQUESTED** [review]

**Review by tristan-orourke — CHANGES_REQUESTED** [review]
I moved the script include, but its still not working. The banner does appear in IE, and not other browsers, but the "Learn more about this decision" button doesn't do anything, and the "Dismiss" button throws a console error: _Object doesn't support property or method 'remove'_

**Comment by mnigh** [comment]
> I moved the script include, but its still not working. The banner does appear in IE, and not other browsers, but the "Learn more about this decision" button doesn't do anything, and the "Dismiss" button throws a console error: _Object doesn't support property or method 'remove'_

This works for me with a previous version of Hydrogen (v0.0.32). @bitliege: Is there anything that has changed with the Dialog component since v0.0.32?

When I switch out the version in the [twig template file](https://github.com/GCTC-NTGC/TalentCloud/blob/feature/ie-alert/resources/views/common/scripts.html.twig#L23) from the following, it works again:

https://cdn.jsdelivr.net/npm/@hydrogen-design-system/system@latest/dist/compiled/latest/system.min.js
to
https://cdn.jsdelivr.net/npm/@hydrogen-design-system/system@v0.0.32/dist/compiled/latest/system.min.js

For the route posted to on _Dismiss_, you may need to clear the route cache since **/api/v1/browser-message** is a new route: `php artisan route:clear` (macOS, not sure on Windows how to do this).

File: resources/assets/js/browserMessage.js [hunk 2]
```diff
@@ -0,0 +1,24 @@
+import { axios } from "./api/base";
```
[the remark this answers is not in the document]
Comment (mnigh) [reply resources/assets/js/browserMessage.js:1]: @tristan-orourke I'm not entirely sure, but I did this based on the [alerts.js](https://github.com/GCTC-NTGC/TalentCloud/blob/feature/ie-alert/resources/assets/js/alerts.js#L1) that does the same thing with axios.

**Comment by tristan-orourke** [comment]
Okay, with #e89645899ffbcda1934ad4587aeda2b7ec5b4fa9 the dismiss button now works.

I was going to suggest that we set the cookie directly in the `browserMessage.js` file, to avoid an extra request and fairly pointless api endpoint, but then I realized Laravel does some nice encryption for us that we may not want to skip.

File: resources/assets/js/browserMessage.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (tristan-orourke) [reply resources/assets/js/browserMessage.js:1]: Seems to work.

**Comment by tristan-orourke** [comment]
I hate to suggest this now after so much work... but I now feel like we should just make the banner non-dismissible. We really don't expect the site to work in IE at this point. Thoughts? @mnigh @bitliege

**Comment by mnigh** [comment]
> I hate to suggest this now after so much work... but I now feel like we should just make the banner non-dismissible. We really don't expect the site to work in IE at this point. Thoughts? @mnigh @bitliege

I agree. The purpose of the banner is to alert the user that the site that they are currently visiting will not work using the browser they are currently using: dismissing the banner won't change that fact and they should not continue using the site.

**Comment by mnigh** [comment]
Resolves #4579.

**Review by yonikid15 — APPROVED** [review]
Nice work 👍 Just missed removing the goc and alert on one file. I had the branch open so I just did it quickly.

**Review by yonikid15 — APPROVED** [review]

**Review by tristan-orourke — APPROVED** [review]

**Pull request merged**